assets/javascripts/modules/checkout/submit.js

```diff
@@ -78,18 +84,20 @@ define([
                 }
             }
         });
-
-
     }
 
-    function send() {
+    function send(ophanPageViewId) {
         var form = formEls.$CHECKOUT_FORM[0];
         var data = serializer([].slice.call(form.elements));
         var submitEl = formEls.$CHECKOUT_SUBMIT[0];
         var errorElement = $('.js-error');
 
         errorElement.text('');
 
+        data.ophanBrowserId = cookie.getCookie('bwid');
```

Comment:
Have you checked that the subscribe.theguardian.com site sets these cookies if visited not via any other theguardian.com domain? Right now subs includes the membership.js Ophan tag, but it might get inserted after this .send?